### PR TITLE
test/status.bats: improve

### PR DESCRIPTION
**Pull request by kolyshkin**
#### What type of PR is this?

/kind cleanup
> /kind dependency-change
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

#### What this PR does / why we need it:

Improvements for `status.bats` tests:

* Remove excessive `run` usage
* Move `echo` to `run_crio_status`
* Fix shellcheck SC2086 warning
* Reformat using shfmt (which gained bats support recently, see https://github.com/mvdan/sh/issues/600)

#### Which issue(s) this PR fixes:

None

#### Special notes for your reviewer:

Please review commit-by-commit. Last one is just whitespace changes.

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Review by saschagrunert — APPROVED** [review]
LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM
and happy green test buttons

File: test/status.bats [hunk 1]
```diff
@@ -3,102 +3,95 @@
...
-
-    # when
-    run_crio_status --socket="${CRIO_SOCKET}" containers --id "$ctr"
-    echo "$output"
-
-    # then
-    [ "$status" -eq 0 ]
-    [[ "$output" == *"sandbox: $pod"* ]]
+	# given
+	run crictl runp "$TESTDATA"/sandbox_config.json
```
Comment (umohnani8) [inline test/status.bats:63]: Don't we want to remove the `run` here and in the following few lines as well?

Comment (kolyshkin) [reply test/status.bats:63]: Yes. I lost that commit during the rebase :-\

Let me redo this

Comment (kolyshkin) [reply test/status.bats]: done; thanks for noticing this!

**Pull request merged**